plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableDataStoreTests.php

```diff
@@ -844,6 +844,86 @@ public function test_cot_query_pagination(): void {
 		$this->assertEquals( array_slice( $test_orders, 5, 5 ), $query->orders, 'The expected dataset is supplied when paginating through orders.' );
 	}
 
+	/**
+	 * Test the `get_order_count()` method.
+	 */
+	public function test_get_order_count(): void {
+		$no_orders_by_status = array(
```

Comment:
Optional—it's pretty subjective and perhaps just a thing in my brain and nobody else's—but this might be clearer is the variable name was `$number_of_orders_by_status` or just `$orders_by_status` to keep it short.

(On first reading, I thought `no` meant *no* whereas I think we are using the abbreviation for number?)

Reply:
Makes sense!